templates/mcollective/mcollective-client.cfg.erb

```diff
@@ -3,8 +3,15 @@
 main_collective = mcollective
 collectives = mcollective
 libdir = <%= scope.lookupvar('::openshift_origin::params::ruby_scl_path_prefix') %>/usr/libexec/mcollective
+<% if scope.lookupvar('::openshift_origin::ose_version') != "" -%>
+# logger_type = file cannot be set for OpenShift Enterprise
+# # https://bugzilla.redhat.com/show_bug.cgi?id=963332
+logger_type = console
+loglevel = warn
+<% else -%>
```

Comment:
Sorry, I inverted the conditional in my head here. So this would be enabled only for OSE. 

I think if the fix works for origin as well we can go ahead and fix it for origin too. The other items in your previous pull requests that were places where variable names differed between Origin M4 and OSE 2.2 so we had no option but to conditionalize it.